internal/pkg/mapper/variables/load.go

```diff
@@ -0,0 +1,36 @@
+package variables
+
+import "github.com/keboola/keboola-as-code/internal/pkg/model"
+
+func (m *variablesMapper) MapAfterRemoteLoad(recipe *model.RemoteLoadRecipe) error {
+	// Variables are used by config
+	apiObject, ok := recipe.ApiObject.(*model.Config)
+	if !ok {
+		return nil
+	}
+
+	// Variables ID is stored in configuration
+	variablesIdRaw, found := apiObject.Content.Get(model.VariablesIdContentKey)
+	if !found {
+		return nil
+	}
+
+	// Variables ID must be string
+	variablesId, ok := variablesIdRaw.(string)
+	if !ok {
+		return nil
+	}
+
+	// Create relation
+	internalObject := recipe.InternalObject.(*model.Config)
+	internalObject.AddRelation(&model.VariablesFromRelation{
+		Source: model.ConfigKeySameBranch{
+			ComponentId: model.VariablesComponentId,
+			Id:          variablesId,
+		},
+	})
+
+	// Remove variables ID from configuration content
+	internalObject.Content.Delete(model.VariablesIdContentKey)
```

Comment:
A zmazeme kluc z obsahu konfiguracie, ... v internej reprezentacii.

`mapper.relations`, z prev PRs sa postara o to, ze sa vytvori aj druha strana relacie:
https://github.com/keboola/keboola-as-code/tree/main/internal/pkg/mapper/relations

https://github.com/keboola/keboola-as-code/blob/aa281381d39bd5b8ad808849936a0e9181e065b0/internal/pkg/mapper/relations/load.go#L37-L58